src/nvim/message.c

```diff
@@ -743,7 +743,7 @@ void ex_messages(exarg_T *eap)
   msg_hist_off = TRUE;
 
   s = os_getenv("LANG");
-  if (s != NULL && *s != NUL)
+  if (!s)
```

Comment:
Uck, I've made several one-off errors in this PR, but the unit tests caught none of them. Fixed, but on closer inspection, why do we only print "Message Maintainer: ..." if `$LANG` happens to be set, and not use it?